notebooks/Medical Image Generation with Generative AI.ipynb

```diff
@@ -89,7 +89,10 @@
     "cs_bucket = \"<bucket/container name to push the experiment job data to>\"  # Please fill it with the actual bucket name\n",
     "\n",
     "# Job configuration\n",
-    "timeout = 3600                                                            # Time (in seconds) to wait for a job to be completed"
+    "timeout = 3600                                                            # Time (in seconds) to wait for a job to be completed\n",
+    "\n",
+    "# Please leave the following advanced parameter as None\n",
```

Comment:
This is a bit confusing. How about saying: `None` will use the default value (N) here?

Reply:
update it. thanks